src/process.c

```diff
@@ -683,6 +685,12 @@ cc_shim_launch (struct cc_oci_config *config,
 			args[i++] = g_strdup ("-w");
 		}
 
+		if (i > SHIM_ARG_COUNT) {
+			g_critical("args index exceeds SHIM_ARG_COUNT: %d > %i",
+				SHIM_ARG_COUNT, i);
```

Comment:
shouldn't it be `"...: %d > %d", i, SHIM_ARG_COUNT);` ? (btw, %d and %i are the same for printf)